spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -610,8 +624,31 @@ function recvPacket(
     abortTransactionUnless(connection !== null)
     abortTransactionUnless(connection.state === OPEN)
 
-    abortTransactionUnless(packet.timeoutHeight === 0 || getConsensusHeight() < packet.timeoutHeight)
-    abortTransactionUnless(packet.timeoutTimestamp === 0 || currentTimestamp() < packet.timeoutTimestamp)
+    switch channel.order {
```

Comment:
I imagine the logic like so:
```
if packet.TimedOut() {
    switch channel.Order {
    case ORDERED_ALLOW_TIMEOUT:
        // write timeout reciept
    default:
        abort()
    }   
}
```
Main request is that logically, we are only performing certain logic if the packet has timed out